modules/lib/lib-event/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "lib-event",
+  "version": "1.0.0",
+  "description": "Type definitions for lib-event.",
+  "types": "event.d.ts",
+  "files": [
+    "event.d.ts"
+  ],
+  "scripts": {
+    "build:js": "tsc",
+    "build:types": "tsc --declaration --emitDeclarationOnly --outDir ./",
+    "build": "npm run build:js & npm run build:types",
+    "packlist": "npx npm-packlist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/enonic/xp.git#master"
+  },
+  "keywords": [
+    "enonic",
+    "enonic-xp",
+    "lib-event",
+    "event",
+    "types",
+    "typescript"
+  ],
+  "author": {
+    "name": "Mikita Taukachou",
+    "email": "edloidas@gmail.com",
+    "url": "https://edloidas.github.io/"
+  },
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/enonic/xp/issues"
+  },
+  "homepage": "https://github.com/enonic/xp/tree/master#readme",
+  "devDependencies": {
```

Comment:
I would consider adding *eslint* and *prettier* now in the transition to TypeScript

Example how we do it in XP/TypeScript-projects: https://github.com/ItemConsulting/xp-ntb-import

Reply:
Sure. As I said, ESLint and prettier will definitely be added a bit later. Right now code is to unstable and constantly changes.